src/ui-identity/usercard-popover.directive.html

```diff
@@ -1,5 +1,5 @@
 <span class="clb-usercard-popover"
       popover-append-to-body="vm.popover.appendToBody"
-      popover-is-open="vm.popover.isOpen"
+      popover-is-open="vm.popover.isOpen"popover-trigger="outsideClick"
```

Comment:
https://angular-ui.github.io/bootstrap/#/popover

outsideClick is a trigger supported by uib-popover directive